src/components/pages/UserPage.js

```diff
@@ -31,10 +32,36 @@ export const UserPage = ({ match }) => {
     }
   }, []);
 
+  const onSortChange = (event) => {
+    const sortValue = event.target.value;
+    setSortType(sortValue);
+  };
+
+  const SelectDropdown = () => (
+    <Flex>
+      <Select
+        borderColor="#808080"
+        color="#808080"
+        fontSize="15px"
+        onChange={onSortChange}
+        value={sortType}
+        w="160px"
+      >
+        <option value="lastUpdated">Last Updated</option>
+        <option value="alphabetical">Alphabetical</option>
+        <option value="numStars">Star Count</option>
+      </Select>
+    </Flex>
+  );
+
   return (
     <>
-      <UsernameSearchField size="small" />
-      <RepoList repos={currentUserRepos} user={user} />
+      <Flex align="center" justify="space-between" mt="25px" w="550px">
+        <Heading fontSize="28px">{user}'s Repositories</Heading>
+        <SelectDropdown />
+      </Flex>
+
+      <RepoList repos={currentUserRepos} sortType={sortType} user={user} />
```

Comment:
The Repo List is just a component for the list of repos, while the other elements are brought up into this page